lib/lotus/cli.rb

```diff
@@ -1,110 +1,134 @@
+# TODO: Clarify/Questions
+# Create a migration along with the entity?
+# Reneame generate model => generate entity?
+# Naming slice/apps. I'd keep the slice generator, drop the app generator (slice can do everything app can)
```

Comment:
I don't think `container` is the right term for that, there should be only 1 container with multiple slices.